src/OnlineSales/Migrations/20230703042743_DbFileStorage.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace OnlineSales.Migrations
+{
+    /// <inheritdoc />
+    public partial class DbFileStorage : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "files",
```

Comment:
@Mixer57 table name must be in singular form, i.e. "file"

Reply:
fixed